main.go

```diff
@@ -857,3 +899,4 @@ func EnableLog() {
 	dlog.SetFlags(logFlags)
 	dlog.SetOutput(os.Stderr)
 }
+
```

Comment:
This extra line isn't needed.

Reply:
should be removed - probably added by formatting the code. Have to dig into my settings there